llmproxy.config.yml

```diff
@@ -57,5 +57,6 @@ provider_settings:
     max_output_tokens: 256
     models:
       - text-bison
+      - gemini-pro
```

Comment:
I thought there were 3 models available. Is gemini-pro the only model available?

Reply:
![image](https://github.com/llm-proxy/llm-proxy/assets/46412618/49af0277-1c24-4f89-bca8-69785e0e1ae7)
https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models

Reply:
gemini-pro is the only model available. do we want vision?